helm/chaos-mesh/templates/chaos-daemon-daemonset.yaml

```diff
@@ -21,8 +21,9 @@ spec:
         app.kubernetes.io/name: {{ template "chaos-mesh.name" . }}
         app.kubernetes.io/instance: {{ .Release.Name }}
         app.kubernetes.io/component: chaos-daemon
-    {{- with .Values.chaosDaemon.podAnnotations }}
       annotations:
+        rollme: {{ randAlphaNum 5 | quote }}
```

Comment:
What's this annotation for? I do not see any reference in codes. 🙈

Reply:
It's used to restart the `chaos-daemon` after upgrade/reinstall helm chart, so that it can reload the newly generated certificate